aten/src/ATen/native/native_functions.yaml

```diff
@@ -9323,6 +9323,22 @@
   python_module: special
   variants: function
 
+- func: special_psi(Tensor self) -> Tensor
+  python_module: special
+  variants: function
+
+- func: special_psi.out(Tensor self, *, Tensor(a!) out) -> Tensor(a!)
+  python_module: special
+  variants: function
+
+- func: special_digamma(Tensor self) -> Tensor
```

Comment:
@mruberry @krshrimali 

Can we have just one alias `special_digamma` and use it in `special.h` and `__init__.py`?

`special.h`
```c++
inline Tensor psi(const Tensor& self) {
  return torch::special_digamma(self);
}

inline Tensor& psi_out(Tensor& result, const Tensor& self) {
  return torch::special_digamma_out(result, self);
}

inline Tensor digamma(const Tensor& self) {
  return torch::special_digamma(self);
}

inline Tensor& digamma_out(Tensor& result, const Tensor& self) {
  return torch::special_digamma_out(result, self);
}
```
And similarly for `__init__.py`
```python
psi = add_docstr(special_digamma, "")
digamma = add_docstr(special_digamma, "")
```

Does this make sense? Will it affect JIT in any way?

Reply:
I think that will work fine and I don't think there's any restriction that an operation have at most one alias.

Reply:
I tried doing this but this will throw an error for documentation, as there will already be a docstr for `special_digamma`:

```
Traceback (most recent call last):
  File "/home/krshrimali/git/krshrimali/pytorch/test/test_ops.py", line 4, in <module>
    import torch
  File "/home/krshrimali/git/krshrimali/pytorch/torch/__init__.py", line 694, in <module>
    from torch import special as special
  File "/home/krshrimali/git/krshrimali/pytorch/torch/special/__init__.py", line 40, in <module>
    digamma = _add_docstr(_special.special_digamma,
RuntimeError: function 'special_digamma' already has a docstring
```

Post offline discussion with @kshitij12345 - we think that the current implementation is probably OK. @mruberry - What do you think?

Reply:
Interesting. Thanks for checking that out. Too bad.